index.html

```diff
@@ -16,84 +16,122 @@
 </head>
 
 <body>
+<div class="container">
   <div class="container-fluid sticky-top">
```

Comment:
Container-fluid sticky-top needs to be outside the container class so that it will wrap everything in that container, like nav, etc. Then, it should stick to the top from the start of the scroll.